src/back/routes/notification.ts

```diff
@@ -30,6 +32,17 @@ async function sendNotification(req: WithAuth) {
     throw new RequestError('Invalid data', RequestError.BadRequest)
   }
 
+  const users = await UserModel.getDiscordIdsByAddresses([recipient])
```

Comment:
Shouldn't this be `[...recipient]`? I think recipient could be string or string[].